english/bite-the-marshmallow/tasty-marshmallow.md

```diff
@@ -17,4 +17,7 @@ Wait a minute....how did you end up in this car?  You must be hallucinating
 
 [Ask the driver "Where are we going?!"](ricky/ricky.md)
 
-[Procrastinate with social media](../lamping/lamping.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.